### DISCUSSION TITLE
Length of the citation key displayed in BetterBibTex?

### DISCUSSION THREAD

**Discussion by earthmoon** [retorquere/zotero-better-bibtex#2656] [Q&A] [answered]
Hello,

This is a citaiton key `americanpublichealth2017a`, but in the right panel of the **Zotero**, only `americanpublichealth2017` is displayed.

Please see this
![image](https://github.com/retorquere/zotero-better-bibtex/assets/45897471/775c8f33-5620-44a6-a9a8-c13158c7950e)

The last character (**a**) is missing.

**Comment by retorquere** [comment]
Making it wider messes with the rest of the item box layout. This is something Zotero is going to address in the upcoming version 7.

**Reply by earthmoon** [reply]
Thanks.

**Reply by retorquere** [reply]
I've found a fix in #2699

**Reply by earthmoon** [reply]
But with the version of Better BibTex for Zotero 6.7.136, there is still that problem.

**Reply by retorquere** [reply]
Can you get me a new screenshot?

**Comment by earthmoon** [comment]
> Can you get me a new screenshot?

![image](https://github.com/retorquere/zotero-better-bibtex/assets/45897471/ef29267a-00ce-465c-92ab-f218261d47c6)

On Debian 12.

**Reply by retorquere** [reply, marked as answer]
Does an upgrade to .137 fix it? I might have just not released it.

**Reply by earthmoon** [reply]
Where can I download .173?

**Reply by retorquere** [reply]
You can go into Tools -> Addons -> Gear menu -> Check for updates

**Reply by earthmoon** [reply]
`No updates found` is returned.

**Reply by retorquere** [reply]
And you're sure you're still on .136?

**Reply by earthmoon** [reply]
Thanks. I have updated it successfully and now it displays well.
